api/config/testdata/valid-imagebuilder-nodeselectors.yaml

```diff
@@ -0,0 +1,3 @@
+ImageBuilderConfig:
+  NodeSelectors:
+    "cloud.google.com/gke-nodepool": image-build-job-node-pool
```

Comment:
Not really a comment on the PR but with the changes I don't think you need the quotes for `cloud.google.com/gke-nodepool` anymore.